packages/inventory-general-info/.gitignore

```diff
@@ -1,6 +1,21 @@
-/index.js
-/index.css
-/TextInputModal.css
-/TextInputModal.js
-/esm
-/cjs
+# ignore everything in root that will be the destination of build output
+/*
+
+# white list allowed files
+!src
+!src/
+!src/*
+!config
+!config/
+!config/*
+!.npmignore
+!.gitignore
+!babel.config.js
+!LICENSE
+!package.json
+!README.md
+!tsconfig.json
+!pre-publish.js
+
+# Incremental ts info
+tsconfig.tsbuildinfo
```

Comment:
I am started feeling this should be a common file. :thinking:

Reply:
Yep. Once we migrate I will create a common gitignore for all packages. I am adding some entries in each migration. Same goes for the babel config and some scripts.